create/create.js

```diff
@@ -0,0 +1,11 @@
+import { addPost } from '../fetch-utils.js';
+
+const newPostEl = document.querySelector('.new-post-form');
+const homeButtonEl = document.querySelector('.home-button');
+
```

Comment:
There should be a redirect here, back to auth page, for users that aren't logged in.